CHANGELOG.md

```diff
@@ -1,278 +1,279 @@
 # Unreleased
 
-* Feat: Support maxSpan for performance API and expose SentryOptions through Hub (#716)
-* Fix: await ZonedGuard integration to run (#732)
-* Fix: `sentry_logging` incorrectly setting SDK name (#725)
+- Feat: Support maxSpan for performance API and expose SentryOptions through Hub (#716)
```

Comment:
Roll back the changes from `*` to `-` since this is the already defined way.

Reply:
I am sorry for that, my linter played me :(